src/configuration.ts

```diff
@@ -77,6 +77,12 @@ const configuration = {
     get buildArguments(): string[] {
         return vscode.workspace.getConfiguration("swift").get<string[]>("buildArguments", []);
     },
+    /** Environment variables to set when building */
+    get swiftEnvironmentVariables(): { [key: string]: string } {
```

Comment:
I guess we can extract `Object.keys(swiftEnvironmentVariables).length > 0` in to a new computed property?

Reply:
Not sure this is totally necessary, I do the same in execFile to make it more obvious why we are setting runtimePath env.